### PR TITLE
fixes #235 (slight cognitive dissonance in the "Roles and Entities" section)

### LINKED ISSUE #235: slight cognitive dissonance in the "Roles and Entities" section
This sentence:
```
   An entity in the RATS architecture includes at least one of the roles
   defined in this document.
```
creates a bit of cognitive dissonance.

In the Terminology section, the Endorser, Ref-val Provider, RP Owner and Verifier Owner are defined as "entities", however, they don't include any of the roles (Verifier, RP, Attester).

### PR DESCRIPTION
**Pull request by henkbirkholz**
* harmonized phrasing in the terminology section -> roles are finally roles and
* changed Terminology item format

### CODE REVIEW

**Comment by henkbirkholz** [comment]
Fixes #235

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -529,76 +529,95 @@ This document uses the following terms.
...
 : A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation.
-Produces: Evidence.
+
+: Produces: Evidence
 
 Relying Party:
 
-: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}.
-Consumes: Attestation Results.
+: A role performed by an entity (typically a device) that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:537]:
```suggestion
: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}.
```

Disagree that it's "typically a device".  What it typically is varies by use case.  Such a statement is fine on a per use case basis, but not in the definition of Relying Party in my opinion.

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -529,76 +529,95 @@ This document uses the following terms.
...
+
+: Consumes: Attestation Results
 
 Verifier:
 
-: A role performed by an entity that appraises the validity of Evidence about an Attester
+: A role performed by an entity (typically a device) that appraises the validity of Evidence about an Attester
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:543]:
```suggestion
: A role performed by an entity that appraises the validity of Evidence about an Attester
```

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -529,76 +529,95 @@ This document uses the following terms.
...
 
 Claim:
-: A piece of asserted information, often in the form of a name/value pair. Claims make up the usual structure of Evidence.
+
+: A piece of asserted information, often in the form of a name/value pair. Claims make up the usual structure of Evidence and other RATS artifacts.
 Compare /claim/ in {{RFC7519}}.
 
 Endorsement:
 
 : A secure statement that an Endorser vouches for the integrity of an Attester's various capabilities such as Claims collection and Evidence signing
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:583]:
```suggestion
: A secure statement that an Endorser vouches for the integrity of an Attester's various capabilities such as Claims collection and Evidence signing.
```

**Pull request merged**